### DISCUSSION TITLE
the best code editor to use for beginners?

### DISCUSSION THREAD

**Discussion by Hartwell9White** [community/community#125142] [Copilot Conversations] [answered]

**Comment by PerezChris99** [comment, marked as answer]
i highly recommend vs code

**Comment by PerezChris99** [comment]
You can download it here 
https://code.visualstudio.com/download

**Comment by ebndev** [comment]
Hi @Hartwell9White & @PerezChris99, thanks for being a part of the GitHub Community!

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
